defaultStyles.css

```diff
@@ -0,0 +1,34 @@
+/** 
+  * defaultStyles.css
```

Comment:
looks like naming css files should be all lowercase

Reply:
Will make changes accordingly